tests/integration/workflows/fixtures/oai_arxiv_record_with_plots.xml

```diff
@@ -24,7 +24,7 @@
             </author>
           </authors>
           <title>Spectra generated by a confined softcore Coulomb potential</title>
-          <categories>math-ph math.MP hep-ex</categories>
+          <categories>math-ph math.MP</categories>
```

Comment:
? Uh? It's not a hack. It's that the `hep-ex` category should not be ingested (as of now), but this fixture had been created for a successful ingestion (unfortunately the wrong example of record had been picked up).